networking/networking.go

```diff
@@ -83,8 +84,18 @@ func Setup(podRoot string, podID types.UUID, fps []commonnet.ForwardedPort, netL
 		},
 	}
 
+	err := os.MkdirAll(mountNetnsDirectory, 0755)
+	if err != nil {
+		return nil, err
+	}
+
+	err = n.mountNetnsDirectory(false)
```

Comment:
We were not doing this before, and perhaps if it doesn't succeed we don't need to hard-fail. Should we perhaps just log an error if the mount fails here and keep going? @squeed?